crouton.php

```diff
@@ -5,7 +5,7 @@
 Description: A tabbed based display for showing meeting information.
 Author: bmlt-enabled
 Author URI: https://bmlt.app
-Version: 3.26.1
+Version: 3.26.2
```

Comment:
this has to be 4.0.1 now as you pushed 4.0.0 tag again (im guessing by accident because you had it locally) and it released to wordpress

Reply:
Thanks, corrected.